readme.md

```diff
@@ -1,6 +1,8 @@
 > This is a list of must-watch videos devoted to JavaScript
 
 ## 2015
+1. [Andre Staltz: What if the user was a function?](https://www.youtube.com/watch?v=1zj7M1LnJV4) [32:19]
+1. [Sufian Rhazi: Transform your codebase](https://www.youtube.com/watch?v=X8WgkMbEAVQ) [33:49]
```

Comment:
Yup, you're right. Sufian's talk was very informative and a good introduction into these tools for me but perhaps not a "must-watch". I saw the talk you linked as well and thanks for the reference. The thing I actually liked about Sufian's talk was that it was more about putting the tools to use in a particular context than about the tools themselves which is what I get from the Babel talk.

As an example, I had known about factories, decorators, etc for a long time but after I saw the "Testable Javascript" talk on the list is when I started putting it together i.e. the practical uses of those talks. Do you know what I mean?

So feel free to leave this talk out. You are doing a great job with balancing the talks here between practical use and technical detail.